index.php

```diff
@@ -66,7 +67,7 @@
                     <p class="card-text mb-auto">Each year, one student is chosen from the ranks of the club to be the sole driver and primary caretaker of the Ramblin’ Reck.</p>
                     <a href="/reck/drivers.php">Read more</a>
                 </div>
-                <img class="card-img-right img-fluid flex-auto d-none d-md-block" data-src="/img/2021/rosman.jpg" style="width: 200px; height: 250px;" src="/img/2021/rosman.jpg" data-holder-rendered="true">
+                <img class="card-img-right img-fluid flex-auto d-none d-md-block" data-src="/img/2021/pie.jpg" style="width: 200px; height: 250px;" src="/img/2021/rosman.jpg" data-holder-rendered="true">
```

Comment:
You need to check for the session here, because otherwise this will be visible to the public

Reply:
Ah thanks for spotting that! I made an edit to the wrong block of code